### PR TITLE
CB-3000 - cloudStorage request validation in SDX and Cloudbreak

### PR DESCRIPTION
**Pull request by attilapalfi92**

### CODE REVIEW

File: datalake/src/main/java/com/sequenceiq/datalake/service/sdx/SdxService.java [hunk 1]
```diff
@@ -171,6 +176,29 @@ public SdxCluster createSdx(final String userCrn, final String name, final SdxCl
         return sdxCluster;
     }
 
+    private void validateCloudStorageRequest(SdxCloudStorageRequest cloudStorage) {
+        if (cloudStorage != null) {
+            ValidationResultBuilder validationBuilder = new ValidationResultBuilder();
```
Comment (doktoric) [inline datalake/src/main/java/com/sequenceiq/datalake/service/sdx/SdxService.java]: can we also check that the baselocation contains the protocol ?  @attilapalfi92

Comment (attilapalfi92) [reply datalake/src/main/java/com/sequenceiq/datalake/service/sdx/SdxService.java]: For the telemetry (LOG identity) it's not necessary. I'm checking if it's necessary for SDX (IDBroker identity)

Comment (doktoric) [reply datalake/src/main/java/com/sequenceiq/datalake/service/sdx/SdxService.java]: cool thanks

Comment (attilapalfi92) [reply datalake/src/main/java/com/sequenceiq/datalake/service/sdx/SdxService.java]: added the validation for SDX

**Pull request merged**